core_data_modules/traced_data/io.py

```diff
@@ -106,3 +107,54 @@ def import_coda_to_traced_data_iterable(user, data, key_of_raw, key_of_coded, f,
             td.append_data({key_of_coded: code}, Metadata(user, Metadata.get_call_location(), time.time()))
 
             yield td
+
+
+class TracedDataCSVIO(object):
+    @staticmethod
+    def export_traced_data_iterable_to_csv(data, f, headers=None):
+        """
+        Writes a collection of TracedData objects to a CSV.
+        
+        Columns will be exported in the order declared in headers if that parameter is specified,
+        otherwise the output order will be arbitrary.
+
+        :param data: TracedData objects to export.
+        :type data: iterable of TracedData
+        :param f: File to export to, opened in 'wb' mode.
+        :type f: file-like
+        :param headers: Headers to export. If this is None, all headers will be exported.
+        :type headers: list of str
+        """
+        data = list(data)
```

Comment:
Can we check to ensure this was an iterable, as before

Reply:
This isn't going to work (because TracedData is iterable). 
A better alternative would be to assert that every item in data is an instance of TracedData.